core/templates/dev/head/domain/suggestion/SuggestionThreadObjectFactorySpec.ts

```diff
@@ -18,28 +18,55 @@
 
 // TODO(#7222): Remove the following block of unnnecessary imports once
 // SuggestionThreadObjectFactory.ts is upgraded to Angular 8.
-import { SuggestionObjectFactory } from
-  'domain/suggestion/SuggestionObjectFactory';
 import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
 require('domain/suggestion/SuggestionThreadObjectFactory.ts');
 
 describe('Suggestion thread object factory', function() {
+  var SuggestionThreadObjectFactory = null;
```

Comment:
I think you can move this to the beforeEach block, and avoid a separate function to check it each time.

Reply:
I'm little confused, are you refering to SuggestionThreadObjectFactory or suggestionThreadBackendDict variable?

Reply:
I meant the suggestionThreadBackendDict, I might have marked the wrong line while commenting. Sorry about the confusion!